tests/integration/targets/aci_l3out_bgp_peer/tasks/main.yml

```diff
@@ -1499,7 +1500,373 @@
         - remove_vpc_bgp_peer.current == []
         - remove_vpc_bgp_peer.previous.0.bgpPeerP.attributes.dn == "uni/tn-ansible_tenant/out-ansible_l3out/lnodep-ansible_node_profile/lifp-ansible_interface_profile/rspathL3OutAtt-[topology/pod-1/protpaths-201-202/pathep-[ansible_vpc_ipg]]/peerP-[192.168.50.2]"
         - remove_vpc_bgp_peer.previous.0.bgpPeerP.attributes.addr == "192.168.50.2"
+  # ADD dynamic BGP peer (prefix) to ethernet port
+  - name: add dynamic BGP peer (prefix) to ethernet port (version >= 4)
+    cisco.aci.aci_l3out_bgp_peer:
```

Comment:
could you set an anchor and reuse in the task that follow, this way it is less code and easier to see what is being updated